### PR TITLE
Fixed existing acceptance tests

### PR DESCRIPTION
**Pull request by chaitra-mylarappachar**
- Fixed existing acceptance tests.
  Moved only acceptance test files under the internal/acceptance_test folder and unit tests will remain with their go files. 
- Replaced Quattro with Metal
- Updated README with instructions for running tests
- Enabled unit tests in CI flows

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,3 +36,39 @@ $ make build
...
+
+2. Set the environment variable HPEGL_METAL_GL_TOKEN to `false` to indicate Metal authentication mode.
+```bash
+export HPEGL_METAL_GL_TOKEN=false
+```
+
+3. Run test
+```
+make acceptance
+```
```
Comment (neilgierman) [inline README.md:74]: In `make acceptance` you are using the `-cover` parameter. Should we have follow up steps here on what to do with that coverage data?

File: internal/acceptance_test/resource_network_test.go [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
 
 import (
 	"fmt"
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
-
-	"github.com/hewlettpackard/hpegl-metal-terraform-resources/pkg/configuration"
+	"github.com/hewlettpackard/hpegl-metal-terraform-resources/pkg/client"
```
Comment (neilgierman) [inline internal/acceptance_test/resource_network_test.go:11]: Blank line between?

File: README.md (hunk 1 above)
Comment (chaitra-mylarappachar) [reply README.md:74]: removed -cover option.

**Comment by chaitra-mylarappachar** [comment]
> I think you need to configure your formatter to treat github.com/hewlettpackard as local for import formatting.

Added gci & goimports section in the lint config file. Hopefully, this will catch such errors going forward.

**Review by neilgierman — APPROVED** [review]

**Pull request merged**